test/unit/spec/common/cellComponents/tabs/jobStatusTable-Spec.js

```diff
@@ -117,82 +117,89 @@ define([
         },
     };
 
-    const paramTestsJobArray = [
-            {
-                job_id: 'job_update_test',
-                status: 'created',
-                created: 50,
-                retry_parent: 'generic_retry_parent',
-                batch_id: 'batch_job',
-                meta: {
-                    jobAction: 'cancel',
-                    jobLabel: 'queued',
-                    retryTarget: 'generic_retry_parent',
-                    row_id: 'generic_retry_parent',
-                },
-            },
-            {
-                job_id: 'generic_retry_parent',
-                status: 'terminated',
-                batch_id: 'batch_job',
-                created: 10,
-            },
-            {
-                batch_id: 'batch_job',
-                job_id: 'batch_job',
-                batch_job: true,
-                child_jobs: ['generic_retry_parent', 'job_update_test'],
-            },
-        ],
-        paramTests = [
-            {
-                input: {
-                    app_id: 'kb_uploadmethods/import_fastq_interleaved_as_reads_from_staging',
-                    app_name: 'Import FASTQ',
-                    batch_id: null,
-                    job_id: 'something',
-                    job_params: [
-                        {
-                            name: 'some_old_file',
-                            workspace_id: 'some_workspace_id',
-                        },
-                    ],
-                },
-                type: 'FASTQ Reads Interleaved',
-                object: /Reads Object Name: some_old_file/,
-                displayData: paramDisplayData.fastq_reads_interleaved,
-            },
-            {
-                input: {
-                    app_id: 'kb_uploadmethods/import_fasta_as_assembly_from_staging',
-                    app_name: 'Import FASTA',
-                    batch_id: null,
-                    job_id: 'something',
-                    job_params: [
-                        {
-                            assembly_name: 'assembly_file.fa',
-                            staging_file_subdir_path: '/path/to/dir',
-                            type: 'draft isolate',
-                            workspace_id: 'whatever',
-                        },
-                    ],
-                },
-                type: 'Assembly',
-                object: /Assembly object name: assembly_file.fa/,
-                displayData: paramDisplayData.assembly,
-            },
-        ],
-        jobUpdateTestJob = {
-            job_id: 'job_update_test',
+    const BATCH_PARENT_ID = 'batch_parent',
+        TEST_JOB_ID = 'job_update_test_job',
+        RETRIED_JOB_ID = 'generic_job_retry',
+        RETRY_PARENT_ID = 'generic_retry_parent';
```

Comment:
I did a fair bit of renaming in this file whilst thinking about how I would need to update the tests for upcoming changes to message handling. There has not been any content or functionality change, other than a couple of places where the `addListener` function arguments needed to be updated.